### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by alexAubin**

### CODE REVIEW

**Comment by alexAubin** [comment]
!testme

File: doc/DISCLAIMER.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+## YunoHost specific features
+
+In addition to Roundcube core features, the following are made available with this package:
+
+ * Synchronize your email aliases as identities in Roundcube
+ * Install the [contextmenu](https://plugins.roundcube.net/packages/johndoh/contextmenu) and [automatic addressbook](https://plugins.roundcube.net/packages/sblaisot/automatic_addressbook) plugins by default
+ * Allow to install the [CardDAV](https://plugins.roundcube.net/packages/roundcube/carddav) (address book) synchronization plugin at the installation - note that if you have installed Nextcloud or Baïkal, it will automatically add the corresponding and existing address book.
+* Support for PGP encryption with Enigma plugin by default.
+
```
Comment (alexAubin) [inline doc/DISCLAIMER.md:13]: I think I'd move this to DESCRIPTION.md ? ... Not a big deal though I guess

**Comment by ericgaspar** [comment]
!testme

File: doc/DESCRIPTION_fr.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+## Caractéristiques spécifiques YunoHost
+
+En plus des fonctionnalités principales de Roundcube, les éléments suivants sont disponibles avec ce paquet :
+
+ * Synchronisez vos alias de messagerie en tant qu'identités dans Roundcube.
+ * Installation des plugins [contextmenu](https://plugins.roundcube.net/packages/johndoh/contextmenu)
+   et [automatic addressbook](https://plugins.roundcube.net/packages/sblaisot/automatic_addressbook) par default.
+ * Permettre d'installer [CardDAV](https://plugins.roundcube.net/packages/roundcube/carddav) (carnet d'adresses) de synchronisation à l'installation - notez que si vous avez installé Nextcloud ou Baïkal, il ajoutera automatiquement le carnet d'adresses correspondant.
+* Prise en charge du cryptage PGP avec le plugin Enigma installé par default.
```
Comment (alexAubin) [inline doc/DESCRIPTION_fr.md:11]:
```suggestion
* Prise en charge du chiffrement PGP avec le plugin Enigma installé par default.
```

#nitpicking :stuck_out_tongue:

**Comment by alexAubin** [comment]
Lets merge it then o/

**Pull request merged**